### ISSUE TITLE
Text does not Align correctly when i convert txt file to image

### ISSUE THREAD

**Issue by jsellin78** [ImageMagick/ImageMagick#5887]
### ImageMagick version

7.1

### Operating system

Linux

### Operating system, version and so on

Debian 10

### Description

Text does not align correctly when i convert a txt file to imag

 See Picture Below.

### Steps to Reproduce

I've googled alot about this issue, haven't found any solutions yet after weeks of searching.

Command i've tried, 

convert -define pango:markup=true -font "DejaVu-Sans-Mono-Bold" -background skyblue -size 600x600 -flatten label:@/tmp/out8.txt result29.png

### Images

![align](https://user-images.githubusercontent.com/34406807/208213233-95ba818f-f320-450f-bfeb-915ce7559247.jpg)

**Comment by urban-warrior** [comment]
Post a link to your text file, `out8.txt`.  We need to download it and reproduce the issue before we can comment further.

**Comment by jsellin78** [comment]
> Post a link to your text file, `out8.txt`. We need to download it and reproduce the issue before we can comment further.

Thank you for the quick reply i really appreciate it. Here is link to out7 txt file.

https://pastebin.com/P8YKmXrF

**Comment by urban-warrior** [comment]
We get expected results.  The text columns align due to a mono font.  We're using ImageMagick version 7.1.0-54.  Add `-debug annotate` to your command-line.  Inspect the debugging output to see if there were any font substitutions.  We get the following:

```
  Font /usr/share/fonts/dejavu-sans-mono-fonts/DejaVuSansMono-Bold.ttf; font-encoding none; text-encoding none; pointsize 16
2022-12-17T01:21:41+00:00 0:00.299 0.340u 7.1.0 Annotate convert[1779217]: annotate.c/RenderFreetype/1751/Annotate
```

**Comment by jsellin78** [comment]
Did you get the same result as i did?
Is there a possibility to put some command in the textfile to tell imagemagic to keep it formatted as it is? 

So maby the more Monospaced fonts i try the better chance of it aligning correctly in the end?

Yeah is says clearly fond encoding none and text-encoding none so maby i need to install a specific addon or util 

Thanks alot for the tip

**Comment by urban-warrior** [comment]
Recall, we claimed the columns of text are aligned.  Your columns are not-- suggesting you are not using a mono font.  That is why we suggested there may be font substitution and to find out with the debug annotate option.  The debugging will tell you if a mono-font was selected.  Maybe you don't have DejaVuSansMono installed or perhaps ImageMagick cannot find it.  The debugging will alert you to any issues with fonts.
